de.metas.shipper.gateway.dhl/src/main/java/de/metas/shipper/gateway/dhl/DhlDeliveryOrderRepository.java

```diff
@@ -313,6 +342,33 @@ private I_DHL_ShipmentOrderRequest toCreateShipmentOrderRequestPO(@NonNull final
 					shipmentOrder.setDHL_Shipper_CountryISO3Code(pickupAddress.getCountry().getAlpha3());
 				}
 
+				//				{
```

Comment:
again, please clean this up, or add a comment explaining the use of this commented out code